SimTracker/TrackTriggerAssociation/plugins/TTClusterAssociator.h

```diff
@@ -53,17 +53,13 @@ class TTClusterAssociator : public edm::stream::EDProducer<> {
 private:
   /// Data members
   edm::Handle<edm::DetSetVector<PixelDigiSimLink> > thePixelDigiSimLinkHandle;
-  edm::Handle<std::vector<TrackingParticle> > TrackingParticleHandle;
+  edm::Handle<std::vector<TrackingParticle> > trackingParticleHandle;
```

Comment:
member variables should end in underscore

Reply:
done